src/OutgoingMessage.ts

```diff
@@ -85,24 +88,28 @@ export default class OutgoingMessage extends NativeOutgoingMessage {
         ? reasonOrHeaders
         : headersOrUndefined;
 
+    // eslint-disable-next-line no-underscore-dangle, no-invalid-this
     if (this._headers && headers !== undefined) {
       // Slow-case: when progressive API and header fields are passed.
       Object.keys(headers).forEach(k => {
         const v = headers[k];
         if (v) {
+          // eslint-disable-next-line no-invalid-this
           this.setHeader(k, v);
         }
       });
     }
 
     // 4. Sets everything
+    // eslint-disable-next-line no-invalid-this
     this.updateResponse(res => ({
       ...res,
       // In order to uniformize node 6 behaviour with node 8 and 10,
       // we want to never have undefined headers, but instead empty object
       headers:
+        // eslint-disable-next-line no-underscore-dangle, no-invalid-this
         this._headers && headers === undefined
-          ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          ? // eslint-disable-next-line no-underscore-dangle, @typescript-eslint/no-explicit-any, no-invalid-this
             (this as any)._renderHeaders()
```

Comment:
`_renderHeaders` is not defined by us nor is in the public specification of `NativeOutgoingMessage`.